src/index.ts

```diff
@@ -7,9 +7,9 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace Cypress {
     interface Chainable {
-      usePactWait: (alias: AliasType) => Chainable
+      usePactWait: (alias: AliasType, providerState?: string) => Chainable
       usePactRequest: (option: AnyObject, alias: string) => Chainable
-      usePactGet: (alias: string, pactConfig: PactConfigType) => Chainable
+      usePactGet: (alias: string, pactConfig: PactConfigType, providerState?: string) => Chainable
```

Comment:
I'm not sure `pactConfig` should have been here. it isn't shown in the examples and wasn't used in line 39 until this change.